src/Core/Scheduler/Providers/WMS_Provider.js

```diff
@@ -7,6 +7,78 @@
 import * as THREE from 'three';
 import Extent from '../../Geographic/Extent';
 import OGCWebServiceHelper from './OGCWebServiceHelper';
+import Fetcher from './Fetcher';
+
+const mimeTypeByTagName = {
+    JPEG: 'image/jpeg',
+    PNG: 'image/png',
+    GIF: 'image/gif',
+    PPM: 'image/x‑portable‑pixmap',
+    TIFF: 'image/tiff',
+    GeoTIFF: 'image/tiff',
+    WebCGM: 'image/cgm;Version=4;ProfileId=WebCGM',
+    SVG: 'image/svg+xml',
+    WMS_XML: 'application/vnd.ogc.wms_xml',
+    'GML.1': 'application/gml+xml',
+    'GML.2': 'application/gml+xml',
+    'GML.3': 'application/gml+xml',
+    WBMP: 'image/vnd.wap.wbmp',
+    MIME: 'www/mime', // Not sure of this one (there's also message/rfc822)
+};
+/* For reference, there are also these 2 possible tags that do not translate to mimetype
+   INIMAGE display text in the returned image -->
+   BLANK return an image with all pixels transparent if
+   */
+
+function _parseSupportedCrs(version, xmlLayer) {
+    let supportedCrs = [];
+    const crsPropName = version === '1.3.0' ? 'CRS' : 'SRS';
+
+    for (const childElem of xmlLayer.children) {
```

Comment:
Searching for items by tag name is often used. It could be factoring.

Otherwise, it will not be easier to transform the xml object js?
I also found lib js that turns the capes into json.

Reply:
> Searching for items by tag name is often used. It could be factoring.

I've hesitated to do so, but each time it's a bit different. So I decided against it because:
- It will only abstract the for loop in the general case (the if part is always a bit different)
- it is less readable than the current code. And it's better to optimize for the reader of our code.

Reply:
> Otherwise, it will not be easier to transform the xml object js?

In my opinion, not worthy the extra lib. It won't save me the for loop anyway, and it adds a layer of abstraction that doesn't bring a lot of value here.

Reply:
To elaborate, as we already have everything we need to query and parse xml in js, I think we can do without. It will avoid growing itowns.js.

Reply:
ok